.github/workflows/create-vsix-artifact.yml

```diff
@@ -48,7 +49,10 @@ jobs:
         env: 
           GOOGLE_OAUTH_CLIENT_SECRET: ${{ secrets.GOOGLE_OAUTH_CLIENT_SECRET }}
           GOOGLE_OAUTH_CLIENT_ID: ${{ secrets.GOOGLE_OAUTH_CLIENT_ID }}
-        run: yarn build:patch:local:ci
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+        run: |
+          echo "DENDRON_RELEASE_VERSION=`cat ./packages/plugin-core/package.json | jq ".version" -r | awk -F. -v OFS=. 'NF==1{print ++$NF}; NF>1{$NF=sprintf("%0*d", length($NF), ($NF+1)); print}'`" >> $GITHUB_ENV
```

Comment:
heads up that as part of the build process we generate a `meta.json` file in `packages/plugin-core` that looks like the following. you can use this to get the release version

```json
{"@dendronhq/api-server":"0.58.3","@dendronhq/common-all":"0.58.3","@dendronhq/common-frontend":"0.58.3","@dendronhq/common-server":"0.58.3","@dendronhq/common-test-utils":"0.58.3","@dendronhq/dendron-11ty":"1.58.3","@dendronhq/dendron-cli":"0.58.3","dendron-design-system":"0.1.0","@dendronhq/dendron-next-server":"0.58.3","@dendronhq/engine-server":"0.58.3","@dendronhq/engine-test-utils":"0.58.3","generator-dendron":"0.1.0","@dendronhq/lsp-client-uber":"0.28.7-alpha.0","@dendronhq/lsp-server":"0.28.7-alpha.0","@dendronhq/plugin-core":"0.58.3","@dendronhq/pods-core":"0.58.3"}
```